mattermost/mattermost-webapp
# MM-20339 add E2E for "Can change Enable Custom Branding setting"

Summary
Test the ability to change "Custom Branding setting" setting

Test steps:

Login as a sysadmin and visit the page "/admin_console/site_config/customization" directly
Verify that setting is visible and matches text content
Verify that help setting is visible and matches text content
Set EnableCustomBrand checkbox to true
Verify that the value is saved, directly via REST API
Ticket Link
Jira ticket [MM-20339](https://mattermost.atlassian.net/browse/MM-20339)
closes https://github.com/mattermost/mattermost-server/issues/13169

Related Pull Requests
Related PR - #4210
